docs/strategy-advanced.md

```diff
@@ -11,6 +11,52 @@ If you're just getting started, please be familiar with the methods described in
 !!! Tip
     You can get a strategy template containing all below methods by running `freqtrade new-strategy --strategy MyAwesomeStrategy --template advanced`
 
+## Packages
+
+It's possible to use python packages as well, so this folder structure is valid too:
+
+```
+user_data/
+├── AwesomeStrategy
+│   └── __init__.py
+└── various_strategies
+│   ├── __init__.py
+    ├── advanced
+    │   ├── advanced_a.py
+    │   ├── advanced_b.py
+    │   └── __init__.py
+    └── simple
+        └── __init__.py
```

Comment:
This structure is wrong and missing the `strategy` level. 
We'll not be searching outside of the strategy directory (unless `--strategy-path user_data/` is defined).